partcad-cli/src/partcad_cli/click/commands/test.py

```
@@ -0,0 +1,144 @@
+
+        asyncio.run(
+            cli_test_async(
+                ctx,
+                packages,
+                filter_prefix=filter,
+                sketch=sketch,
+                interface=interface,
+                assembly=assembly,
+                scene=scene,
+                object=object,
+            )
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)